tests/test_manylinux.py

```diff
@@ -9,7 +9,7 @@
 
 VERBOSE = True
 ENCODING = 'utf-8'
-MANYLINUX_IMAGE_ID = 'quay.io/pypa/manylinux1_x86_64'
```

Comment:
Shouldn't the tests using manylinux1 still be run ?

Reply:
I've updated the tests to run against the manlinux1 and manylinux2010 containers.